tests/new_api_test.cpp

```diff
@@ -1345,6 +1345,260 @@ void test_remove_corrupted(const ioremap::elliptics::newapi::session &session) {
 	BOOST_REQUIRE_EQUAL(result.status(), 0);
 }
 
+void test_bulk_read(const ioremap::elliptics::newapi::session &session) {
+	record record{
+		std::string{"key"},
+		0xff1ff2ff3,
+		dnet_time{10, 20},
+		dnet_time{10, 20},
+		std::string{"{\"key\": \"key\"}"},
+		512,
+		std::string{"key data"},
+		1024,
+		(session.get_ioflags() & DNET_IO_FLAGS_CACHE) != 0
+	};
+
+	typedef std::tuple<int, key> key_t; // (group_id, key)
+	std::map<key_t, tests::record> records;
+	std::set<key_t> responses;
+
+	auto check_all_responses_presence = [&records, &responses] () {
+		for (auto it : records) {
+			const auto &key = it.first;
+			BOOST_REQUIRE_EQUAL(responses.find(key) == responses.end(), false);
```

Comment:
It is because iterator can't be streamed to std::ostream, so use `BOOST_REQUIRE(A != B)` instead.